package.json

```diff
@@ -35,8 +35,8 @@
     "octopus:deploy": "octopus-deploy-create-release-and-deploy",
     "octopus:publish": "gulp octopus-publish",
     "postinstall": "link-parent-bin",
-    "publish:all": "lerna publish --conventional-commits --message \"build: publish\"",
-    "publish:all:beta": "yarn publish -- --npm-tag=beta",
+    "publish:all": "yarn build && yarn publish:all --ignore-scripts --ignore-prepublish",
```

Comment:
this is the main change. we no longer are running `prepublish` in each package. now we run a single build from the root for all packages.

we're also using the `ignore-prepublish` option to ignore any prepublish scripts in the packages